### ISSUE TITLE
Add a check for any enzymatic parent-child GO relations that have inconsistent EC xrefs

### ISSUE THREAD

**Issue by pgaudet** [geneontology/go-ontology#23286]
Hi @balhoff 

Can we add a check for any enzymatic parent-child GO relations that have inconsistent EC xrefs? E.g. GO parent term has EC:1.5.1.- but GO child term has EC that doesn't match EC:1.5.1.*

Thanks, Pascale

**Comment by cmungall** [comment]
Yes, we should do this for everything we map to that has a hierarchy, incl rhea. Jim we have code somewhere for building obo files for many of these sources
